### ISSUE TITLE
[1.11.2] Progression not saved

### ISSUE THREAD

**Issue by Terpo** [lorddusk/HQM#330]
Currently I'm testing the latest 1.11.2 version of hqm (HQM-New Heights-1.11.2-5.1.0alpha3) on a server / client environment. 

I'm unable to create a team. 
Client log for this: https://gist.github.com/Terpo/8cfea4535157951861620bf09b666b14
Server log for this: Nothin in latest and nothing in forge log at this timestamps

Also, without a team, when I'm finishing a quest and restart the server, all progression is reset.

Config on client:
sync with server: true
no quests when connecting first time

Config on server:
sync with server: true
has all quests 

The syncronization itself works

**Comment by Dominance** [comment]
Does your client / server have edit mode on. Even if server has it disabled and client has it enabled. you will be unable to make a team. and visa versa.

**Comment by Terpo** [comment]
no, don't have edit mode enabled on server and client.
client:
editmode.cfg https://gist.github.com/Terpo/54bc32adc7dbe5b78bb2b0fe02300d98
hqmconfig.cfg https://gist.github.com/Terpo/0fb9d3f14b0042b07ea998a9af3b208a

server:
editmode.cfg https://gist.github.com/Terpo/1dc12506a4252c781cc1b7afd1c89b4a
hqmconfig.cfg https://gist.github.com/Terpo/51bb1e524bfcdc01c8d3516fa7b15e35

**Comment by Dominance** [comment]
At ok. I just knew that to be another cause as to why you cannot make parties.

**Comment by Terpo** [comment]
I did some more testing with a completly fresh server<->client enviroment.
First: creation of teams does work, but the client has to disconnect and reconnect to the server to get synchronized data. After reconnect the player is in the team.

Second: The problem with not saving does not seem to be a problem of the mod itself, because with just JEI and HQM it is working as intended. All I can say at this point is, that one of my categories or quests is broken and for that, the server can't save the data. I start trying a new quest book on the whole modpack to see if it could be a cross-mod problem. 

edit: Tested with my whole modpack and a new book --> everything works. 
So this means that somewhere in my 200 quests is a problem with saving. I will try to find out which category works and which not.

One more confirmation: It is not a server<->client problem here. Just switch client to editmode false and tested the first quest. After reloading the world the quest was reset :/.
